### PR TITLE
move to 0.5

### PR DESCRIPTION
**Pull request by alexcos20**

### CODE REVIEW

File: plecos/samples/algorithm_metadata_remote.json [hunk 1]
```diff
@@ -6,7 +6,6 @@
     "datePublished": "2019-02-08T08:13:49Z",
     "license": "CC-BY",
     "name": "My super algorithm",
-    "price": "1",
```
Comment (0x3bfc) [inline plecos/samples/algorithm_metadata_remote.json:9]: if not price, is it a `dtCost` ?

File: tests/test_validate.py [hunk 2]
```diff
@@ -17,17 +17,16 @@ def test_validator_simple():
     schema = {
         "type": "object",
         "properties": {
-            "price": {"type": "number"},
             "name": {"type": "string"},
         },
-        "required": ["price", "name"]
+        "required": ["name"]
```
Comment (0x3bfc) [inline tests/test_validate.py:22]: dtCost, name

File: plecos/samples/algorithm_metadata_remote.json (hunk 1 above)
Comment (alexcos20) [reply plecos/samples/algorithm_metadata_remote.json:9]: plecos only does metadata section, and not the services section.  there is no reason to have a dtCost for metadata

File: tests/test_validate.py (hunk 2 above)
Comment (alexcos20) [reply tests/test_validate.py:22]: plecos only does metadata section, and not the services section.  there is no reason to have a dtCost for metadata

**Pull request merged**